habitat/config/default.py

```diff
@@ -17,7 +20,8 @@ def __init__(self, *args, **kwargs):
 
 CN = Config
 
-DEFAULT_CONFIG_DIR = "configs/"
+_HABITAT_CFG_DIR = osp.dirname(osp.abspath(__file__))
```

Comment:
What is the advantage compare to `osp.dirname(inspect.getabsfile(inspect.currentframe())` as it was in previous PR?

Reply:
Actually it should include parent folder otherwise paths `habitat/configs/` doesn't make sense.

Reply:
> What is the advantage compare to osp.dirname(inspect.getabsfile(inspect.currentframe()) as it was in previous PR?
no need to `import inspect` and `__file__` is more understandable than `currentframe()` IMO.

> Actually it should include parent folder otherwise paths habitat/configs/ doesn't make sense.
I don't get it, `_HABITAT_CFG_DIR=habitat/config`

Reply:
I wrote that piece with inspect. IIRC, `__file__` wasn't working correctly with all install types.

Reply:
Strange, we use `__file__` in a bunch of different places. 
I can change it here, no problem

Reply:
> Actually it should include parent folder otherwise paths `habitat/configs/` doesn't make sense.

To be clear, if the user creates a `configs` directory and puts a yaml in there. Calling `habitat.get_config("configs/my_config.yaml")` will work if called from the root directory of `configs` (because relative path works). I think this should be the recommended way to create/modify environments for users who do not want to clone the entire repo but still want to create custom environments.